library/src/main/java/com/novoda/downloadmanager/Download.java

```diff
@@ -3,11 +3,17 @@
 public class Download {
 
     private final long id;
+    private final String title;
+    private final String description;
+    private final int status;
     private final long currentSize;
     private final long totalSize;
 
-    public Download(long id, long currentSize, long totalSize) {
+    public Download(long id, String title, String description, int status, long currentSize, long totalSize) {
```

Comment:
no, he doesn't :-1: - method name says `modifyQueuedOrDownloadingNotification` but if we want to add more (say you want to customise the notification for completed downloads) then we'd have to add more methods which is not ideal IMO - or if you want to differentiate between `Queued` and `Downloading` at the moment you can't